docs/connecting-via-dotnet.adoc

```diff
@@ -0,0 +1,46 @@
+[[connecting-via-dotnet]]
+== Connecting from .NET
+
+JanusGraph can be queried from a .NET application with Apache TinkerPop's http://tinkerpop.apache.org/docs/{tinkerpop_version}/reference/#gremlin-DotNet[Gremlin.Net].
+
+Gremlin traversals can be constructed with Gremlin.Net just like in Gremlin-Java or Gremiln-Groovy. Refer to <<gremlin>> for an introduction to Gremlin and pointers to further resources. The main syntactical difference for Gremlin.Net is that it follows .NET naming conventions, e.g., method names use PascalCase instead of camelCase.
+
+=== Getting Started with JanusGraph and Gremlin.Net
+
+To get started with Gremlin.Net:
+
+. Create a console application:
+[source, bash]
+dotnet new console -o GremlinExample
+
+. Add Gremlin.Net:
+[source, bash]
+dotnet add package Gremlin.Net -v $MAVEN{tinkerpop.version}
+
+. Create a `GraphTraversalSource` which is the basis for all Gremlin traversals:
++
+[source, csharp]
+----
+var graph = new Graph();
+var client = new GremlinClient(new GremlinServer("localhost", 8182));
+// The client needs to be disposed on shut down to release resources and close open connections
```

Comment:
In the python example you actually have a comment saying to use connection.close(). I think it would be nice to have a similar comments for .NET and Java. 

Also you used `close` instead of `dispose` in the other two examples. Not sure if dispose is more semantically correct for .NET, but there wasn't much to comment on so I figured I'd include it.

Reply:
I'll add explicit references to the methods that need to be called for Java and .NET.

Regarding `Dispose()` in .NET: This method is part of the `IDisposable` interface. It is basically always used in cases where resources are released on shut down (like open connections, file handles, ...). This allows to use `using` in C# which looks like this:
```c#
using (var client = new GremlinClient(new GremlinServer("localhost", 8182)))
{
    // do something with the client
}  // <-- `client.Dispose()` is called automatically here even if an exception was thrown
```
`Close()` would be more in line with Gremlin-Java, but it wouldn't feel natural to .NET developers and it couldn't be used with `using`.